### DISCUSSION TITLE
PyMechanical with Ansys 2022 R2

### DISCUSSION THREAD

**Discussion by faisalaldebie** [ansys/pymechanical#559] [Q&A] [answered]
### 🔍 Before submitting the issue

- [X] I have searched among the existing issues
- [X] I am using a Python virtual environment

### 🐞 Description of the bug

Is it possible to launch PyMechanical with Ansys version 2022 R2 ?

### 📝 Steps to reproduce

..

### 💻 Which operating system are you using?

Windows

### 📀 Which ANSYS version are you using?

2022 R2

### 🐍 Which Python version are you using?

3.11

### 📦 Installed packages

```shell
pyansys 2024.1.1
```

**Comment by dipinknair** [comment]
We are currently supporting only Mechanical 2023 R1 and later for remote and embedding in windows. 2023 R2 and later for embedding in Linux

https://mechanical.docs.pyansys.com/version/stable/getting_started/versioning.html#pymechanical-remote-sessions

**Comment by dipinknair** [comment, marked as answer]
Support is now available only for 2023 R2 or later.
